### PR TITLE
Adds helm chart

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Towards https://github.com/giantswarm/giantswarm/issues/21334

### CODE REVIEW

File: helm/operational-load-exporter/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+description: A Helm chart for operational-load-exporter
+home: https://github.com/giantswarm/operational-load-exporter
+name: operational-load-exporter
+appVersion: [[ .AppVersion ]]
+version: [[ .Version ]]
```
Comment (pipo02mix) [inline helm/operational-load-exporter/Chart.yaml:6]: why not use app bundle? it makes this automatic for you

**Review by pipo02mix — APPROVED** [review]
LGTM beyond the comment

File: helm/operational-load-exporter/Chart.yaml (hunk 1 above)
Comment (JosephSalisbury) [reply helm/operational-load-exporter/Chart.yaml:6]: @pipo02mix can you show me an example? i'm learning this as i go

Comment (pipo02mix) [reply helm/operational-load-exporter/Chart.yaml]: [Alfred used this one](https://github.com/giantswarm/alfred-app) but the [template-app](https://github.com/giantswarm/template-app) is the reference

**Pull request merged**